src/dafny/DafnyCore.dfy

```diff
@@ -0,0 +1,14 @@
+// RUN: %diff "%s" "%s"
+
+include "./BoundedInts.dfy"
+include "./Results.dfy"
+include "./Collections/Collections.dfy"
+include "./FileIO/FileIO.dfy"
+include "./Math.dfy"
+include "./NonlinearArithmetic"
```

Comment:
Did you mean to create a top-level `.dfy` file for this directory, too? It doesn't seem valid, as-is.

Reply:
I put in the diff commands as No-ops, because lit complains (and fails) if there is no RUN command. But a resolve command would be better. I put the top-level DafnyCore.dfy file, though it has no functionality of its own, in so that a user can include it (as a library) and get the whole library.

I'll change those diffs to resolve where they exist.

Reply:
All of that is fixed up and tests successfully - a few new files.

Reply:
Agreed.